apps/speed.c

```diff
@@ -2037,6 +2118,9 @@ int speed_main(int argc, char **argv)
     /* default iteration count for the last two EC Curves */
     ecdh_c[R_EC_X25519][0] = count / 1800;
     ecdh_c[R_EC_X448][0] = count / 7200;
+
+    eddsa_c[R_EC_Ed25519][0] = count / 1800;
+    eddsa_c[R_EC_Ed448][0] = count / 7200;
```

Comment:
Where do these numbers come from?

Reply:
Frankly I don't know why these numbers, I just saw the comments on top of the X25519 stuffs and set this to EdDSA as a 'default iteration count'. Maybe @romen has more understandings on this?

Reply:
I wish I could say so, but frankly I never really understood completely that part. It's mpstly dead code, for platforms without `SIG_ALERT`.

@mattcaswell 
> Urgh...speed is such a horrible application :-(

I have been sharing the feeling for a long time, can't wait for a proper refactor!

Reply:
These factors comes from some short tests on my computer ;)
See https://github.com/openssl/openssl/pull/6133.

Reply:
Those figures are simply a scaling based on manual comparison based on the performance testing performed with SIG_ALERT isn't available to interrupt. 
It is scaling relative to calls to DES_ecb_encrypt - so how many of this algorithm do we think we can perform relative to the number of DES calls we can perform in 3 seconds. 
Generally you figure out the scaling factor by running the test and seeing the relative execution speeds and then applying the right factor to be close to what you need.